src/data/loader.py

```diff
@@ -192,7 +192,7 @@ def get_events_metadata(
             key=lambda event: (
                 get_date_timestamp(event.stop_date) * sort_order,
                 get_date_timestamp(event.start_date) * sort_order,
-                normalized_key('name'),
```

Comment:
Why aren't you normalising it now?  We need to ensure that names with accents are sorted appropriately.